src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -484,6 +484,15 @@
     crafted: true
 """
 
+helps['backup vault update'] = """
+type: command
+short-summary: Update an existing Recovery Services vault.
+examples:
+  - name: Update immutability settings of a Recovery Services vault.
+    text: az backup vault update --name MyRecoveryServicesVault --resource-group MyResourceGroup --immutability-state Unlocked
+    crafted: true
```

Comment:
`crafted: true` means this help message is generated by Aladdin. if it is a handwritten help message, please remove this tag
```suggestion
```